src/CheckoutSdk/Common/PaymentSourceType.cs

```diff
@@ -24,5 +21,16 @@ public enum PaymentSourceType
         [EnumMember(Value = "rapipago")] RapiPago,
         [EnumMember(Value = "klarna")] Klarna,
         [EnumMember(Value = "sofort")] Sofort,
+        [EnumMember(Value = "knet")] KNet,
+        [EnumMember(Value = "qpay")] QPay,
+        [EnumMember(Value = "alipay")] Alipay,
+        [EnumMember(Value = "paypal")] PayPal,
+        [EnumMember(Value = "multibanco")] Multibanco,
+        [EnumMember(Value = "eps")] EPS,
+        [EnumMember(Value = "poli")] Poli,
+        [EnumMember(Value = "przelewy24")] Przelewy24,
```

Comment:
przelewy24 == p24 so no need to declare this twice https://www.checkout.com/docs/payments/payment-methods/bank-transfers/przelewy24

Reply:
Good catch, will fix it.

Reply:
Applied, thanks!